examples/Baselines/Halite_competition/paddle/rl_trainer/algorithm.py

```diff
@@ -0,0 +1,143 @@
+#   Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Can we use parl.algorithms.PPO directly?

Reply:
Since the action space is discrete, we still need to rewrite the sample, predict and learn methods in parl.algorithms.PPO.  We can use parl.algorithms.a2c instead.

Reply:
Since the action space is discrete, we still need to rewrite the sample, predict and learn methods in parl.algorithms.PPO. I can try dqn based algorithms instead and see if the performance is satisfiable.

Reply:
Got it. It's ok to use the PPO algorithm.